### PR TITLE
Add a way to show xrays for models in collection views

### PR DESCRIPTION
**Pull request by ranquild**
Adds a menu option to x-ray a model from collection views.

How to test:
- Open a collection with a model
- Click on the ellipsis icon -> X-ray this 
- A dashboard should appear

Screenshots:
Screenshot 2023-03-16 at 14 42 50
Screenshot 2023-03-16 at 14 42 54

### CODE REVIEW

File: frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx [hunk 1]
```diff
@@ -28,92 +40,135 @@ const setup = ({
...
 };
 
 describe("ActionMenu", () => {
-  it("should show an option to hide preview for a pinned question", () => {
-    const item = createMockCollectionItem({
-      model: "card",
-      collection_position: 1,
-      collection_preview: true,
-      setCollectionPreview: jest.fn(),
+  describe("preview", () => {
```
Comment (ranquild) [inline frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx:50]: grouped tests here, no other changes

File: frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx [hunk 2]
```diff
@@ -28,92 +40,135 @@ const setup = ({
...
   });
 
-  it("should not allow to move and archive personal collections", () => {
-    const item = createMockCollectionItem({
-      name: "My personal collection",
-      model: "collection",
-      personal_owner_id: 1,
-      setCollection: jest.fn(),
-      setArchived: jest.fn(),
+  describe("x-rays", () => {
```
Comment (ranquild) [inline frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx:137]: new tests

File: frontend/src/metabase/components/EntityItem.jsx [hunk 3]
```diff
@@ -106,6 +113,12 @@ function EntityItemMenu({
           action: onPin,
           event: `${analyticsContext};Entity Item;Pin Item;${item.model}`,
         },
+        isXrayShown && {
+          title: `X-ray this question`,
```
Comment (kulyk) [inline frontend/src/metabase/components/EntityItem.jsx:117]: Missing `t` here

File: frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx [hunk 4]
```diff
@@ -28,92 +40,135 @@ const setup = ({
...
+        setCollectionPreview: jest.fn(),
+      });
 
-    userEvent.click(screen.getByLabelText("ellipsis icon"));
-    userEvent.click(screen.getByText("Don’t show visualization"));
+      setup({ item });
 
-    expect(item.setCollectionPreview).toHaveBeenCalledWith(false);
-  });
+      userEvent.click(screen.getByLabelText("ellipsis icon"));
```
Comment (kulyk) [inline frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx:77]: We can use `getIcon("ellipsis")` here

File: frontend/src/metabase/lib/urls/dashboards.ts [hunk 5]
```diff
@@ -35,3 +35,7 @@ export function embedDashboard(token: string) {
   const siteUrl = MetabaseSettings.get("site-url");
   return `${siteUrl}/embed/dashboard/${token}`;
 }
+
+export function xrayQuestion(id: CardId) {
+  return `/auto/dashboard/question/${id}`;
+}
```
Comment (kulyk) [inline frontend/src/metabase/lib/urls/dashboards.ts:41]: A bit on the fence here: I'd expect `xrayQuestion` to live in `urls/question`, but the URL itself probably leads to `urls/dashboards`

**Review by kulyk — APPROVED** [review]
Looking great! Please fix the translation before merging

File: frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx (hunk 4 above)
Comment (ranquild) [reply frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx]: Fixed

File: frontend/src/metabase/lib/urls/dashboards.ts (hunk 5 above)
Comment (ranquild) [reply frontend/src/metabase/lib/urls/dashboards.ts]: Fixed, moved to questions

File: frontend/src/metabase/components/EntityItem.jsx (hunk 3 above)
Comment (ranquild) [reply frontend/src/metabase/components/EntityItem.jsx]: Great catch! Fixed

**Comment by ranquild** [comment]
Closed as it was never a planned change.

**Pull request closed without merging**